docs: intro sentence for date-dropdown (#1605)
siemens/ix: packages/documentation/docs/controls/buttons/dropdown-button.md

# Dropdown button

<Tags/>
<!-- introduction start -->

> Dropdown buttons reveal a list of actions on click on a button.

Reviewer: ```suggestion
Dropdown buttons reveal a list of actions when selected.
```
